spec/rubocop/formatter/offense_count_formatter_spec.rb

```diff
@@ -20,14 +20,14 @@ module Formatter
         context 'when no offenses are detected' do
           let(:offenses) { [] }
           it 'does not add to offense_counts' do
-            expect { finish }.not_to change { formatter.offense_counts }
+            expect { finish }.not_to(change { formatter.offense_counts })
```

Comment:
@bbatsov I definitely agree with that. It's better not to break this nice DSL: `expect {}.to change {}`
Many RSpec users would want to disable this cop for their specs.
What do you think the best way to handle that?